[#60] Add Example of containerization integrating Kserve using aissemble OIP (#64)
boozallen/aissemble-inference: aissemble-open-inference-protocol-examples/aissemble-oip-kserve/aissemble-oip-kserve-containerization/README.md

> # aiSSEMBLE Open Inference Protocol KServe Containerization Example

Reviewer: ```suggestion
# aiSSEMBLE Open Inference Protocol&trade; KServe Containerization Example
```
we should add the trademark sign after `aiSSEMBLE Open Inference Protocol` whenever it's in the top header of the README files for consistency!

> This example provides simple guideline on how to implement Containerization of Aissemble Open Interface Protocol and deploy on KServe

Reviewer 2: A: Can you change Aissemble to aiSSEMBLE?

> This example provides simple guideline on how to implement Containerization of Aissemble Open Interface Protocol and deploy on KServe

Reviewer: ```suggestion
This example demonstrates how to containerize the aiSSEMBLE Open Interface Protocol and deploy it on KServe
```

## Prerequisites
* This is intended for users who already have set up base infrastructures (CRD, KServe, ingress, cert-manager) and want to make use of docker containerization using [Habushu Containerization](https://github.com/TechnologyBrewery/habushu/blob/dev/examples/habushu-containerize/README.md) to deploy custom model to Kserve.

## Getting Started

> * This Example Consists of 3 Modules:

Reviewer: can we put all the code snippets in `xyz` format with the single quotes? For example, `aissemble-oip-kserve-containerization-deploy`, `aissemble-oip-kserve-containerization-docker`, etc

Reviewer: and let's update all `Kserve`/`kserve` to `KServe` and `aissemble` to `aiSSEMBLE`!
